dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataset/CompleteDataSetRegistration.java

```diff
@@ -86,14 +94,18 @@ public CompleteDataSetRegistration( DataSet dataSet, Period period, Organisation
     }
 
     public CompleteDataSetRegistration( DataSet dataSet, Period period, OrganisationUnit source,
-        CategoryOptionCombo attributeOptionCombo, Date date, String storedBy )
+        CategoryOptionCombo attributeOptionCombo, Date date, String storedBy, User lastUpdatedBy, Date lastUpdated,
+                                        Boolean isCompleted )

```

Comment:
Codestyle

Reply:
Could you provide some more input related to which code style you are referring too?